chia/util/db_wrapper.py

```diff
@@ -163,12 +117,18 @@ async def create(
         log_path: Optional[Path] = None,
         journal_mode: str = "WAL",
         synchronous: Optional[str] = None,
+        foreign_keys: bool = False,
+        row_factory: Optional[Type[aiosqlite.Row]] = None,
```

Comment:
would you mind putting a `*,` somewhere high up here too? I think we have too many parameters now to allow non-named ones